test_all.sh

```diff
@@ -141,6 +141,13 @@ if [[ $USE_OMPSS -eq 1 ]]; then
     done
 fi
 
+if [[ $USE_OMPSS2 -eq 1 ]]; then
+    for t in $basic_types; do
+        ./ompss2/main -steps 9 -type $t
```

Comment:
Could you follow the format of the other tests here? I.e. a double loop over types and kernels. (See above.) If memory bound kernels don't work yet, you can use `compute_kernels` instead.

Also, if OmpSs2 supports multiple task graphs, please add a line that copies the one with `-and` above.